src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -96,6 +100,20 @@ public void resendStaffNotifications(
         }
     }
 
+    @PutMapping("/claim/resend-rpa-notifications")
+    @ApiOperation("Resend notifications for multiple claims")
```

Comment:
As this endpoint only supports citizen and not legal claims, would be nice to add that here for documentation purpose.